{
  context.getDebugger().addBreakpoint(breakpoint,exceptionCondition);
  getMockEndpoint("mock:result").expectedBodiesReceived("Hello World");
  template.sendBody("direct:start","Hello World");
  try {
    template.sendBody("direct:start","Hello Camel");
    fail("Should have thrown exception");
  }
 catch (  Exception e) {
  }
  assertMockEndpointsSatisfied();
  assertEquals(1,logs.size());
  assertEquals("Breakpoint at ThrowException[java.lang.IllegalArgumentException] caused by: IllegalArgumentException[Damn]",logs.get(0));
}
